config/samples/linstorcluster.yaml

```diff
@@ -6,16 +6,18 @@ spec: {}
 #  repository: registry.example.com
 #  nodeSelector:
 #    piraeus.io/satellite: "true"
+#  linstorPassphraseSecret: my-linstor-passphrase
+#  properties:
+#    - name: TcpPortAutoRange
+#      value: "8000"
```

Comment:
Only just noticed this detail which is also in the README: Does this actually work? Shouldn't the value be "8000-9999" or something of that form?

Reply:
Yes, you are right.